src/js/index.js

```diff
@@ -28,6 +29,12 @@ class Chart {
 
   draw( chartOptions ) {
     switch ( chartOptions.type ) {
+      case 'geo-map':
+        shapes.fetch( chartOptions.metadata ).then( shapes => {
+          chartOptions.shapes = shapes[0];
+          this.highchart = new createChart.GeoMap( chartOptions );
+        } );
```

Comment:
Unlike the other charts, the geo map has to fetch shape files before it creates the chart.